style.css

```diff
@@ -1,66 +1,95 @@
 html{
-    font-family: 'Josefin Sans', sans-serif;
-    box-sizing: border-box;
-  }
-  
-  main {
-    background-image: url("https://images.unsplash.com/photo-1518655048521-f130df041f66?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1650&q=80");
-    background-size: 100%;
-    height: 600px;
-    padding-top: 40px;
-    padding-left: 10px;
-  }
-  
-  #welcome {
-    width: 65%;
-    font-size: 55px;
-    color:#b2bec3;
-  }
-  
-  #welcome_subtitle{
-    font-size: 40px;
-    color: #b2bec3;
-    width: 75%;
-  }
-  
-  nav{
-    overflow: hidden;
-    position: fixed;
-    top: 0;
-    width: 100%;
-    background-color: hsla(198, 12%, 73%, 0.25);
-  }
-  
-  li {
-    list-style-type: none;
-    font-size: 30px;
-    color: #218c74;
-    float: left;
-    display: float;
-    padding: 15px;
-  }
-  
-  a {
-    font-size: 30px;
-    color: #218c74;
-    text-decoration: none;
-  }
-  
-  a.visited{
-    font-size: 30px;
-    color: #218c74;
-  }
-  
-  #project_opener {
-    font-size: 35px;
-    color: #b33939;
-    width: 100%;
-    text-align: center;
-  }
-  
-  #project_text {
-    font-size: 22px;
-    width: 100%;
-    text-align: center;
-  
-  }
\ No newline at end of file
+  font-family: 'Josefin Sans', sans-serif;
+  box-sizing: border-box;
+}
+
+main {
```

Comment:
Apply this with a class, avoid styling HTML elements via the element name at all costs